cli/commands/index.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   highlight: require("./highlight"),
   host: require("./host"),
   init: require("./init"),
-  new: require("./new"),
+  // New keyword cannot be used
+  _new: require("./new"),
```

Comment:
You could export directly the functions instead of the entire module. For example, instead of `_new`, export `require("./new").createPackage`. You can avoid this reserved word conflict, and call directly the functions on the tests.

Reply:
```javascript
const { createPacakge } = require("./new");
```

Is also another option

Reply:
Fixed with e37a9e3